tests/chunk-rollover/resources/helm/fluentbit-basic.yaml

```diff
@@ -0,0 +1,85 @@
+extraVolumeMounts:
+- mountPath: /chunks
+  name: chunks-volume
+extraVolumes:
+- name: chunks-volume
+  emptyDir:
+    sizeLimit: 3Mi
```

Comment:
I've set it 3 MB because of [this](https://docs.fluentbit.io/manual/administration/buffering-and-storage#chunks) to see if it will exceed the 2MB

> A Chunk size usually is around 2MB.